src/js/module/main/main.js

```diff
@@ -41,6 +43,7 @@ const MainViewMain = View.extend({
       }, this)
       .on(Const.Event.START_PLANTING, function() {
         this.$el.find('.plantingjs-startbtn').hide();
+        this.$el.toggleClass(IS_PLANTING_CLASS, true);
```

Comment:
Well, not really... Because it's not good idea to replace dynamically `View.el` - [here author of backbone explains why](https://github.com/jashkenas/backbone/issues/546#issuecomment-3604746).